browser/brave_vpn/brave_vpn_service_factory.cc

```diff
@@ -50,6 +56,14 @@ BraveVpnServiceFactory::BraveVpnServiceFactory()
           "BraveVpnService",
           BrowserContextDependencyManager::GetInstance()) {
   DependsOn(skus::SkusServiceFactory::GetInstance());
+
+#if BUILDFLAG(IS_WIN)
+  if (base::FeatureList::IsEnabled(
```

Comment:
see comment below, we can just remove the feature check (but leave the windows guard)

Reply:
done